pkg/web/openapi.go

```diff
@@ -0,0 +1,56 @@
+package web
+
+import (
+	"io/fs"
+	"net/http"
+	"strings"
+	"sync"
+	"text/template"
+
+	"github.com/gin-gonic/gin"
+	"github.com/rs/zerolog/log"
+	"github.com/trisacrypto/envoy/pkg"
+	api "github.com/trisacrypto/envoy/pkg/web/api/v1"
+)
+
+func (s *Server) OpenAPI(c *gin.Context) {
+	var (
+		err        error
+		data       gin.H
+		templates  *template.Template
+		initialize sync.Once
+	)
+
+	initialize.Do(func() {
+		data = gin.H{
+			"Version":     pkg.Version(),
+			"Origin":      s.conf.Web.Origin,
+			"Description": "Envoy",
```

Comment:
Using `Rotational Envoy` here sounds fine to me. It would be a bit more specific than just `Envoy`.